full-service/migrations/2024-05-21-035622_subaddress-only-account/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
```

Comment:
A migration is required because we are adding a new field to Account, in order to allow a clear delineation of the Account's mode